e2e/config/config.go

```diff
@@ -76,6 +78,12 @@ type PolicyAccounts struct {
 	AdminPolicyAccount       Account `yaml:"admin_policy_account"`
 }
 
+// ObserverRelayerAccounts are the accounts used by the observers to interact with gateway contracts in non-EVM chains (e.g. Solana)
+type ObserverRelayerAccounts struct {
+	RelayerAccount0 Account `yaml:"relayer_account_0"`
```

Comment:
Can you explain why we have 2 accounts in the comment? Why not to use `Accounts [2]Account`?

Reply:
I replaced this with `[2]Account` and added description: https://github.com/zeta-chain/node/pull/2673/commits/98f041a755690e1cfd6c94e35f96763be753f6e1